posthog/tasks/tasks.py

```diff
@@ -716,11 +716,8 @@ def calculate_decide_usage() -> None:
         capture_usage_for_all_teams as capture_decide_usage_for_all_teams,
     )
 
-    ph_client = get_ph_client()
-
-    if ph_client:
-        capture_decide_usage_for_all_teams(ph_client)
```

Comment:
@PostHog/team-billing this sends a `decide usage` and `local evaluation usage` event for teams. Is that something we rely on for billing? Trying to figure out if these events should be going to the US & EU regions for any reason or if it's ok to send them to the US only

Reply:
Yes the is VERY important, it's how we get usage and billing for feature flag usage.

Reply:
Needs to go to the specific region because we store ff usage in the respective cloud.  Please be careful changing any of those code.

Reply:
Gotcha. Will rethink this stuff. Will make any changes necessary for billing in a separate PR. I'm conscious we're not reporting Celery exceptions right now